### PR TITLE
ci: add unparam linter

### PR DESCRIPTION
**Pull request by kolyshkin**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 linters:
   enable:
+    - unparam
     - unconvert
     - errorlint
```
Comment (thaJeztah) [inline .golangci.yml:5]: We should sort this list; helps preventing accidental duplicates and makes it easier to find which ones are enabled

File: mountinfo/mounted_linux_test.go [hunk 2]
```diff
@@ -15,252 +15,247 @@ import (
...
+func tMount(t *testing.T, src, dst, fstype string, flags uintptr) {
 	if os.Getuid() != 0 {
 		t.Skip("root required for mounting")
 	}
 
-	err := unix.Mount(src, dst, fstype, flags, options)
+	err := unix.Mount(src, dst, fstype, flags, "")
 	if err != nil {
-		return &os.PathError{Path: dst, Op: "mount", Err: err}
+		t.Helper()
```
Comment (thaJeztah) [inline mountinfo/mounted_linux_test.go:27]: Can you put this one at the start of the function? I think it's the more natural place to put it; also makes sure that the `t.Skip()` shows the right line-number of the test, not the helper.

File: mountinfo/mounted_linux_test.go [hunk 3]
```diff
@@ -15,252 +15,247 @@ import (
...
 	t.Cleanup(func() {
 		if err := unix.Unmount(dst, unix.MNT_DETACH); err != nil {
+			t.Helper()
 			t.Errorf("cleanup: unmount %q failed: %v", dst, err)
 		}
 	})
-	return nil
+}
+
+func tCreateFile(t *testing.T, dir string) string {
```
Comment (thaJeztah) [inline mountinfo/mounted_linux_test.go:38]: This should probably be a `t.Helper()` as well?

File: mountinfo/mounted_linux_test.go (hunk 2 above)
Comment (kolyshkin) [reply mountinfo/mounted_linux_test.go:27]: Thanks! Done.

File: mountinfo/mounted_linux_test.go (hunk 3 above)
Comment (kolyshkin) [reply mountinfo/mounted_linux_test.go:38]: Right.

File: .golangci.yml (hunk 1 above)
Comment (kolyshkin) [reply .golangci.yml]: done

**Comment by kolyshkin** [comment]
Fixed review comments; PTAL @thaJeztah

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**